### ISSUE TITLE
Icon channel in the playlist tab in main page

### ISSUE THREAD

**Issue by heberjeur** [TeamNewPipe/NewPipe#4137]
### Version

-0.19.8 

### Steps to reproduce the bug

Main screen
I have playlist in tab 
I click to the channel icon to access directly to the channel 
Error

### Screenshots/Screen recordings

![20200816_021250](https://user-images.githubusercontent.com/11852634/90324390-98bad880-df6e-11ea-8a3b-663c81d76a0c.jpg)

## Exception
* __User Action:__ ui error
* __Request:__ App crash, UI failure
* __Content Country:__ TN
* __Content Language:__ ar
* __App Language:__ fr
* __Service:__ none
* __Version:__ 0.19.8
* __OS:__ Linux samsung/a50dd/a50:10/QP1A.190711.020/A505FDDU5BTE1:user/release-keys 10 - 29
<details><summary><b>Crash log </b></summary><p>

```
java.lang.IllegalArgumentException: No view found for id 0x7f090126 (org.schabi.newpipe:id/fragment_holder) for fragment ChannelFragment{32deb96} (dff786b3-b9a5-4351-a662-baa6c2acad03) id=0x7f090126}
	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:315)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1187)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1356)
	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1434)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1497)
	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
	at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2169)
	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1992)
	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1947)
	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1849)
	at androidx.fragment.app.FragmentManager$4.run(FragmentManager.java:413)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:237)
	at android.app.ActivityThread.main(ActivityThread.java:7814)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1075)

```
</details>
<hr>

**Comment by wb9688** [comment]
I think this should be fixed by #3892. Could you test whether the APK at https://github.com/TeamNewPipe/NewPipe/projects/17#card-42918515 doesn't crash?

**Comment by heberjeur** [comment]
Tes it works, thanks
